paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -95,9 +96,8 @@ internal abstract class BaseSheetViewModel(
 
     protected var mostRecentError: Throwable? = null
 
-    @VisibleForTesting
-    internal val _isGooglePayReady = savedStateHandle.getLiveData<Boolean>(SAVE_GOOGLE_PAY_READY)
```

Comment:
Can we initialize false, and keep this a boolean?

Reply:
LiveData was being abused here, there is an indeterminate state, i.e. the backing data is null.